frontend/src/app/core/services/auth.service.ts

```diff
@@ -75,7 +77,6 @@ export class AuthService {
     }
 
     private handleAuthError(error: Error) {
-        // eslint-disable-next-line no-alert
-        window.alert(error.message);
+        this.notificationService.showErrorMessage(error.message);
```

Comment:
I believe you can just replace this method with method content

Reply:
Replaced